src/15utility.js

```diff
@@ -272,6 +272,10 @@ utils.isCordova = (function(){
   return (typeof cordova === 'object')
 })();
 
+utils.isReactNative = (function(){
+  return (typeof require('react-native') === 'object')

```

Comment:
simply check if package 'react-native' is available, please let me know if you have another way like 'isCordova', but I cannot find definition of object 'cordova'.